src/API/Router.test.ts

```diff
@@ -1,3 +1,4 @@
+import { getVerifiableClaimSigner } from '@po.et/poet-js'
```

Comment:
Just delete this file and any other dummy tests. Not worth spending time fixing/updating them - they are only there as a hack around coverage with TS.

Reply:
ack, some were deleted from master already.